src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/tests/latest/test_batchai_custom.py

```diff
@@ -187,22 +187,22 @@ def test_batchai_update_fileserver_user_account_settings_using_command_line_when
         with self.assertRaisesRegexp(CLIError, 'Incorrect ssh public key value'):
             _update_user_account_settings(params, 'user', 'my key', None)
 
-    @patch('getpass.getuser')
```

Comment:
This should not need to change because `get_default_admin_username` leverages getpass.getuser. Please revert.